roles/openshift_logging/tasks/generate_configmaps.yaml

```diff
@@ -8,6 +8,11 @@
         src: elasticsearch.yml
         dest: "{{mktemp.stdout}}/elasticsearch.yml"
 
+    - lineinfile: 
```

Comment:
apparently block statements dont support when.  it looks like you can only apply it to the block